findhelp/schema.py

```diff
@@ -0,0 +1,49 @@
+import graphene
+from graphql import ResolveInfo
+from graphene_django.types import DjangoObjectType
+
+from .models import TenantResource
+
+
+class TenantResourceType(DjangoObjectType):
+    class Meta:
+        model = TenantResource
+        only_fields = ('name', 'address', 'website', 'phone_number', 'description')
+
+    latitude = graphene.Float()
+    longitude = graphene.Float()
+    miles_away = graphene.Float()
+
+    def resolve_latitude(self, info: ResolveInfo):
+        if self.geocoded_point is None:
+            return None
+        return self.geocoded_point[1]
+
+    def resolve_longitude(self, info: ResolveInfo):
+        if self.geocoded_point is None:
+            return None
```

Comment:
So technically, given the fact that `TenantResourceType` is _only_ being used in the context of asking for the best resources, latitude/longitude/distance will _never_ be `None` here, so it might be useful to just make them all required, and only make them optional if/when we have queries that can actually return results with null fields.